components/common/homepage/extra/testimonial.jsx

```diff
@@ -0,0 +1,196 @@
+import Slider from 'react-slick'
+import styles from '../comment.module.css'
+import Image from 'next/image'
+import { Col, Row } from 'react-bootstrap'
+
+const Testimonials = () => {
+  const settings = {
+    infinite: true,
+    slidesToShow: 3,
+    slidesToScroll: 1,
+    vertical: true,
+    verticalSwiping: true,
+    autoplay: true,
+  }
+
+  const appreciation = [
+    {
+      content:
+        'I got an A+ in my test! I love your comprehensive and accurate curriculum. Thanks a lot!',
+      name: 'Isabella',
+      dClass: 'JSS 3',
+    },
+
+    {
+      content:
+        'Learning is easier with Afrilearn. I really love this class, you guys are amazing 😘😘😘😘😘😘😘😘😘',
+      name: 'Omoigho Blessing',
+      dClass: 'JSS 2',
+    },
+
+    {
+      content:
+        'Since I began using Afrilearn to teach my Basic Science classes a few months ago, some of my students who used to score 25% now score over 75%. It’s been extremely exciting for them and for me. I love this new easy way of teaching',
+      name: 'Mr. Idris Lawal',
```

Comment:
Please ensure that the space on the ready for success section is reduced as agreed:

![image](https://user-images.githubusercontent.com/10332499/203934518-651f9a52-8ac9-44c2-8f75-1f6b98491f74.png)